source/FreeRTOS_DNS.c

```diff
@@ -368,7 +368,7 @@
          * still needs to be tested. */
         *ppxNetworkBuffer = pxGetNetworkBufferWithDescriptor( uxExpectedPayloadLength +
                                                               uxHeaderBytes,
-                                                              0UL );
+                                                              ( TickType_t ) 0UL );
 
```

Comment:
Can you please search the whole source code for "[0-9]UL" and remove the "L"?
I think that in this case the explicit cast to `TickType_t` will not be necessary.